dist/src/main/arctic-bin/conf/config.yaml

```diff
@@ -3,15 +3,6 @@ ams:
   admin-password: admin
   server-bind-host: "0.0.0.0"
   server-expose-host: "127.0.0.1"
-  refresh-external-catalog-interval: 180000 # 3min
```

Comment:
Change the config in `site/docs/ch/guilds/deployment.md` too.

Reply:
Okay, I will change it.